go/libkb/resolve.go

```diff
@@ -171,6 +171,7 @@ func (r *Resolver) resolveURLViaServerLookup(au AssertionURL, input string, with
 
 	var them *jsonw.Wrapper
 	if them, res.err = ares.Body.AtKey("them").ToArray(); res.err != nil {
+		res.err = NotFoundError{fmt.Sprintf("User not found for query: %q", input)}
```

Comment:
And also `keybase id thurnish41ey` seems to do the right thing.  Whatever codepath KBFS is using here during rekey might have a bug thogh...